pkg/meta/tkv.go

```diff
@@ -1229,32 +969,8 @@ func (m *kvMeta) Fallocate(ctx Context, inode Ino, mode uint8, off uint64, size
 	return errno(err)
 }
 
-func (m *kvMeta) ReadLink(ctx Context, inode Ino, path *[]byte) syscall.Errno {
-	if target, ok := m.symlinks.Load(inode); ok {
-		*path = target.([]byte)
-		return 0
-	}
-	defer timeit(time.Now())
-	target, err := m.get(m.symKey(inode))
-	if err != nil {
-		return errno(err)
-	}
-	if target == nil {
-		return syscall.ENOENT
-	}
-	*path = target
-	m.symlinks.Store(inode, target)
-	return 0
-}
-
-func (m *kvMeta) Symlink(ctx Context, parent Ino, name string, path string, inode *Ino, attr *Attr) syscall.Errno {
-	defer timeit(time.Now())
-	return m.mknod(ctx, parent, name, TypeSymlink, 0644, 022, 0, path, inode, attr)
-}
-
-func (m *kvMeta) Mknod(ctx Context, parent Ino, name string, _type uint8, mode, cumask uint16, rdev uint32, inode *Ino, attr *Attr) syscall.Errno {
-	defer timeit(time.Now())
-	return m.mknod(ctx, parent, name, _type, mode, cumask, rdev, "", inode, attr)
+func (m *kvMeta) readlink(inode Ino) ([]byte, error) {
```

Comment:
Return ENOENT if value is empty, same as in sql.go.
Then in base maybe we don't need to check `target == nil`